openedx/core/djangoapps/programs/utils.py

```diff
@@ -351,4 +351,12 @@ def supplement_program_data(program_data, user):
             # TODO: Currently unavailable on LMS.
             run_mode['marketing_url'] = ''
 
+            certificate_data = certificate_api.certificate_downloadable_status(user, course_key)
+            certificate_uuid = certificate_data.get('uuid')
+            if certificate_uuid:
```

Comment:
@mattdrayer that depends on how widespread use of certificate UUIDs is. I don't have a good sense of this, nor do I have a good idea of how many courses are configured to use HTML certificates. Is it the default for new courses?

I'm under the impression that the UUID-version of the URL is preferred in contexts where the URL might be shared. Is the alternate version of the URL acceptable in cases like these?